### PR TITLE
feat: add telemetry

### PR DESCRIPTION
**Pull request by SgtPooki**
- feat: add telemetry
- fix: allow typescript to resolve esm import

fixes https://github.com/laurentsenta/pl-diagnose/issues/10

### CODE REVIEW

File: frontend/src/index.tsx [hunk 1]
```diff
@@ -1,25 +1,34 @@
...
+        <QueryClientProvider client={queryClient}>
+            <HashRouter>
+                <CommonParamsProvider>
+                    <App />
+                </CommonParamsProvider>
+            </HashRouter>
+        </QueryClientProvider>
+    </React.StrictMode>,
+    document.getElementById('root'),
 );
```
Comment (SgtPooki) [inline frontend/src/index.tsx:32]: oops. will revert

File: frontend/tsconfig.json [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
+        "noFallthroughCasesInSwitch": true,
+        "module": "esnext",
+        "moduleResolution": "nodenext",
+        "resolveJsonModule": true,
+        "isolatedModules": true,
+        "noEmit": true,
+        "jsx": "react-jsx"
+    },
+    "include": ["src"]
 }
```
Comment (SgtPooki) [inline frontend/tsconfig.json:21]: will revert. my bad

File: frontend/src/index.tsx (hunk 1 above)
Comment (SgtPooki) [reply frontend/src/index.tsx]: this was caused by the pre-commit hook which hasn't been run on existing code.

File: frontend/tsconfig.json (hunk 2 above)
Comment (SgtPooki) [reply frontend/tsconfig.json]: this was caused by the pre-commit hook which hasn't been run on existing code.

**Review by djmcquillan — APPROVED** [review]

**Comment by SgtPooki** [comment]
@laurentsenta when will this be deployed once merged

**Review by laurentsenta — APPROVED** [review]

**Pull request merged**